client/store/choosed-screenshot/choosed-screenshot.js

```diff
@@ -0,0 +1,19 @@
+export default function choosedScreenshot(state = { id: undefined, needsToBeShown: false }, action) {
+  switch (action.type) {
+    case ('SHOW_CHOOSED_SCREENSHOT'): {
```

Comment:
this value should be stored in separate constants file and used here as variable